kasa/tests/newfakes.py

```diff
@@ -414,6 +414,9 @@ def light_state(self, x, *args):
             "get_monthstat": None,
             "erase_emeter_state": None,
         },
+        "smartlife.iot.common.anti_theft": {
+            "get_next_action": {"err_code": -2, "err_msg": "member not support"}
```

Comment:
Not that it makes much difference given that this is unlikely used by anyone, but this probably indicates that the antitheft itself is supported but that the `get_next_action` method is not.